go/vt/sqlparser/utils.go

```diff
@@ -118,6 +118,7 @@ func NormalizeAlphabetically(query string) (normalized string, err error) {
 // replaces any cases of the provided database name with the
 // specified replacement name.
 // Note: both database names provided should be unescaped strings.
+// The returned query is formatted canonically (will look different than original query)
```

Comment:
Is this comment still valid? Since you reverted the call to `CanonicalString`?

Reply:
Fixed